src/plugins/GUITestBase/src/tests/common_scenarios/msa_editor/GTTestsMsaEditor.cpp

```diff
@@ -1504,7 +1504,7 @@ GUI_TEST_CLASS_DEFINITION(test_0020_1) {
     GTMouseDriver::click(Qt::RightButton);
 
     // Expected state: UGENE not crashes, deletion is not performed
-    GTWidget::click(os, GTWidget::findWidget(os, "msa_editor_sequence_area"));
+    GTWidget::click(os, GTUtilsMSAEditorSequenceArea::getSequenceArea(os, 0));
```

Comment:
You can make '0' index default for this method, so tests not related to the multiline mode will have less non-related details.